### DISCUSSION TITLE
Absence and Going Forward

### DISCUSSION THREAD

**Discussion by devbisme** [xesscorp/skidl#205] [General]
My apologies. I haven't been tending to SKiDL (or any of my other Github repos) for a while. For the past seven months, I've been providing daycare for my daughter's baby for 60-70 hours/week. I'm going to try to get to some of these issues as time permits. I'm not trying to make excuses; I just wanted to let you know I'm not ignoring you or abandoning this project.

**Comment by Kreijstal** [comment]
thank you, it is appreciated. Take all time you need
